src/app/lib/analyticsUtils/index.js

```diff
@@ -203,3 +203,11 @@ export const getClickInfo = (elem, { service, component, label, type }) => {
 
   return `PUB-[${service}-${component}]-[=${type}]-[${label}]-[${format}]-[]-[]-[]-[${url}]`;
 };
+
+export const getLibraryVersion = platform => {
+  if (platform === 'amp' || onClient()) {
+    return 'simorgh';
+  }
+
+  return 'simorgh-nojs';
```

Comment:
reading this code I'm thinking we can get here if:

- is canonical
- is requested from server

Is that expected?

Reply:
I'm not 100% sure that is expected. I'll need to ask some questions and get back to you

Reply:
@jroebu14 I believe this is expected

Reply:
I've been thinking - maybe this function should always return `'simorgh'` and then [here](https://github.com/bbc/simorgh/blob/4901428d5204d6c44a8da76017dca46c0037dae4/src/app/containers/ATIAnalytics/canonical/index.jsx#L24) we can do `src={atiPageViewUrl.replace('x8=[simorgh]', 'x8=[simorgh-nojs]')}` since it's inside a `<noscript>`.

Reply:
Okay, that'll be a better approach, ensuring that `simorgh-nojs` always get sent from the client.

Reply:
@jroebu14 this has been fixed